### ISSUE TITLE
babelrc with ignore

### ISSUE THREAD

**Issue by bluelovers** [babel/babel#5146]
### file

```
index.js
+source/index.js
+source/test/index.js
+source/test/test2/index.js
```

### output

1. when `ignore = ["index.js"]` will outoput 0 file <=== sure this
2. when `ignore = ["./index.js"]` will outoput 0 file
3. when `ignore = ["/index.js"]` will outoput only `index.js -> dist\index.js`
4. when `ignore = ["\\index.js"]` will outoput only `index.js -> dist\index.js`

**2** maybe should count as bug
**3** & **4** should is bug

----

**_and any way ignore index.js at root when we only use babelrc + babel cli ?_**

**Comment by xtuc** [comment]
Will be fixed as in #5086

**Comment by hzoo** [comment]
Also with https://github.com/babel/babel/pull/5487 prs with ignore in Babel 7
